### PR TITLE
Gracefully handle multiple reboot/shutdown requests

### PR DESCRIPTION
**Pull request by cywang117**
Since HTTP's server.close() is async, there is a slim chance for two instances of /v1/reboot or /v1/shutdown to be processed. If the server is already closed when server.close() is called, the call throws ERR_SERVER_NOT_RUNNING
 which doesn't need to be surfaced to the user.

Change-type: patch

### CODE REVIEW

File: src/device-api/index.ts [hunk 1]
```diff
@@ -92,20 +88,16 @@ export class SupervisorAPI {
 		});
 	}
 
-	public async stop(options?: SupervisorAPIStopOpts): Promise<void> {
+	public async stop(): Promise<void> {
 		if (this.server != null) {
```
Comment (pipex) [inline src/device-api/index.ts:92]: Could we just make `this.server` null in order to ignore multiple calls to `stop()`, we could change the error for a warning on subsequent calls

Comment (cywang117) [reply src/device-api/index.ts:92]: Sure @pipex, changed to that in https://github.com/balena-os/balena-supervisor/commit/bdc57070c873359e38c3221561cfbbd6b755813d

**Comment by pipex** [comment]
Looks good

**Comment by cywang117** [comment]
@balena-ci I self-certify!

**Pull request merged**